src/service/get-all-tags.ts

```diff
@@ -0,0 +1,47 @@
+import { STRAPI_CLIENT } from '../config/api-client';
+import { MetaInfo } from '../interfaces/meta-info';
+import {
+    getAllContentTypesMetaInfo,
+    getAllExpertiseLevelsMetaInfo,
+    getAllL1ProductsMetaInfo,
+    getAllL2ProductsMetaInfo,
+    getAllProgrammingLanguagesMetaInfo,
+    getAllTechnologiesMetaInfo,
+} from '../api-requests/get-all-meta-info';
+
+import {
+    parseMetaInfoResponse,
+    parseMetaInfoResponseForL1,
+} from './get-all-meta-info';
+
+// Similar to getAllMetaInfo, but doesn't filter out tags without content associated.
+export const getAllTags = async (): Promise<MetaInfo[]> => {
```

Comment:
Will this be pointing to Contentstack later on ?

Reply:
Yes it will. I imagine all the `get<CONTENT_TYPE>MetaInfo` methods will point to ContentStack anyway, so there should be much to change here.